### PR TITLE
chore(release): update release hierarchy

### PR DESCRIPTION
**Pull request by akhilerm**
- update release hierarchy
- update repositories which are used for releasing

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

File: contribute/process/release-management.md [hunk 1]
```diff
@@ -71,18 +70,17 @@ Each repository has the automation scripts setup to push the container images to
...
-      - openebs/external-storage
-        - openebs/maya
-          - openebs/velero-plugin
-          - openebs/cstor-csi
-          - openebs/upgrade
-          - openebs/jiva-operator
-            - openebs/jiva-csi
+        - openebs/velero-plugin
+        - openebs/cstor-csi
+        - openebs/upgrade
```
Comment (kmova) [inline contribute/process/release-management.md:83]: q_: Is upgrade tagged by cstor-operators or cstor-csi?

File: contribute/process/release-management.md [hunk 2]
```diff
@@ -27,15 +27,14 @@ Release branches need to be created for the following repositories:
 - openebs/libcstor
 - openebs/cstor
 - openebs/istgt
-- openebs/external-storage
```
Comment (kmova) [inline contribute/process/release-management.md:30]: Can we add these under manual tagging section. Probably with a new message saying, the following repos are being deprecated and will be manually tagged.

File: contribute/process/release-management.md (hunk 1 above)
Comment (akhilerm) [reply contribute/process/release-management.md:83]: cstor-operators.

File: contribute/process/release-management.md (hunk 2 above)
Comment (kmova) [reply contribute/process/release-management.md:30]:
```
openbs/maya
openebs/openebs-k8s-provisioner
```

Comment (akhilerm) [reply contribute/process/release-management.md:30]: Done.

File: contribute/process/release-management.md [hunk 3]
```diff
@@ -27,25 +27,33 @@ Release branches need to be created for the following repositories:
...
 - openebs/Mayastor
 - openebs/linux-utils
-- openebs/rawfile-localpv
 - openebs/monitor-pv
+- openebs/dynamic-nfs-provisioner
+- openebs/device-localpv
+- openebs/openebsctl
+- openebs/monitoring
+
+The following repositories are deprecated and will be manually tagged
```
Comment (kmova) [inline contribute/process/release-management.md:54]: This should go to below .. after line 103.

Comment (kmova) [reply contribute/process/release-management.md:54]:
```suggestion
The following repositories are being deprecated and will be manually tagged
```

**Review by kmova — APPROVED** [review]

**Pull request merged**